README.md

```diff
@@ -6,43 +15,30 @@ The contract responsible for sending/receiving messages from the base layer gate
 
 - [Documentation/Diagrams](./docs/design_docs.md)
 - [Deploying Token Contracts](./docs/deploy_docs.md)
-- [Build From Source](#building_from_source)
-- [Contributing](#contributing)
 - [License](#license)
 
 ### Bridge Message Predicates
 
 This project uses the general contract message relaying script/predicate from the [bridge-message-predicates](https://github.com/FuelLabs/bridge-message-predicates) repo.
 
-## Building From Source
-
-### Dependencies
+## Quickstart
 
-| dep     | version                                                  |
-| ------- | -------------------------------------------------------- |
-| Forc    | [v0.35.0](https://fuellabs.github.io/sway/v0.35.0/introduction/installation.html) |
+### Building Sway
 
-### Building
+Each Sway project uses a `fuel-toolchain.toml` file with pinned versions so `forc` will install the correct versions for you if you do not have them installed.
 
-Build:
+In the root of the repository run the following command to build all the Sway programs.
 
-```sh
+```bash
 forc build
 ```
 
-Run tests:
+### Running Rust Tests
 
-```sh
-cargo test
-```
+After the Sway programs have been built run the following command in the root of the repository.
 
-## Contributing
```

Comment:
Why was this section removed?

Reply:
It doesn't add any value aside from telling you to format code and CI will fail if it's not formatted anyway so they have to format

Reply:
Right, I thought the whole point of having it in the readme is so you don't have to wait until CI finishes to see that you had to run the formatter. This avoids unnecessary "formatting" commits that ruin the last commit message preview in the github code explorer. Has the team been removing the "contributing" section in other repos too?

Reply:
I believe most repos have squashed commits so this shouldn't be an issue.
Once you learn to format your code once you do it everywhere. Having to remember to check the readme or have it enforced by the CI doesn't change anything and since we enforce it via CI it's redundant to remind people.

It's not about removing that section from places it's just that this content doesn't add value here so it was removed

Reply:
I think is better to have the format command than don't have it. It don't hurt to help the devs here.

Reply:
I disagree and that doesn't counter my point; however, since it's not a change that is harmful and there's 2 people that are requesting it now then I'll add it back in soon